website/docs/components/form/super-select/partials/code/component-api.md

```diff
@@ -0,0 +1,198 @@
+  <C.Property @name="searchPlaceholder" @type="string" @default="Search">
+    Displays placeholder text in the search input when `@searchEnabled` is set to `true`.
+  </C.Property>
+  <C.Property @name="selected">
+    The selected option object. Can be used to set a pre-selected option.
+  </C.Property>
+  <C.Property @name="onChange">
+    A closure action that is called when the selected option changes.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### SuperSelect::Single::Field
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="id" @type="string">
+    Super Select Single’s ID attribute.
+    <br/><br/>
+    By default, the ID is automatically generated by the component. Use this argument to pass a custom ID.
+  </C.Property>
+  <C.Property @name="isInvalid" @type="boolean" @default="false">
+    Applies an “invalid” appearance to the control but doesn’t modify its logical validity.
+  </C.Property>
+  <C.Property @name="isRequired" @type="boolean" @default="false">
+    Appends a `Required` indicator next to the label text and sets the `required` attribute on the control when user input is required.
+  </C.Property>
+  <C.Property @name="isOptional" @type="boolean" @default="false">
+    Appends an `Optional` indicator next to the label text when user input is optional.
+  </C.Property>
+  <C.Property @name="width" @type="string" @valueNote="any valid CSS width (px, rem, etc)">
+    By default, `SuperSelect` has an intrinsic width of 100%. If a `@width` parameter is provided, the control will have a fixed width. This width will only be applied to the control, not the other elements of the field.
+  </C.Property>
+  <C.Property @name="extraAriaDescribedBy" @type="string">
+    Extra ID attribute to add to the `aria-describedby` HTML attribute.
+    <br/><br/>
+    By default, the `aria-describedby` attribute is automatically generated by the component, using the IDs of the helper text and errors (if present). Use this argument to pass an extra ID.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+    <br/><br/>
+    The attributes will be applied to the `SuperSelect` component. This means you can use all the standard HTML attributes and all the usual Ember techniques for event handling, validation, etc.
+    <!-- <br/><br/>
+    Examples of HTML attributes: `name`, `placeholder`, `disabled`, `readonly`, `required`. See [the whole list of HTML attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes). Examples of Ember modifiers: `\{{on "input" [do something]}}`, `\{{on "change" [do something]}}`, `\{{on "blur" [do something]}}`. -->
+  </C.Property>
```

Comment:
Not yet sure what it makes sense to include here so I commented it out for now (and also for the `Multiple::Field`) ...